primary/views/layout.pug

```diff
@@ -44,7 +44,7 @@ html(lang=locale dir=getLocaleDirection(locale))
 				2022: '2022-grid.svg',
 				2023: '2023-halftone.svg',
 				//2024: '2024-film-texture.webp'
-				2024: 'green_brush_strokes_vertical.jpg'
+				2024: 'FIRST_IN_SHOW_FRC_CRESCENDO_Wallpaper_Dark_cleanup_lg.jpg'
```

Comment:
make sure new files match the naming convention, so `2024-paintbrush` & see if webp has better filesize than jpg
might also be good to fix 2020 while you're at it